mxnet/training/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -97,8 +108,8 @@ WORKDIR /
 
 RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
  && pip install --no-cache --upgrade \
-    # Protobuf above this version appears to result protobuf errors.
-    protobuf==3.20.1 \
+    # Protobuf >3.20 results in protobuf errors version mismatch errors with framework and other packages.
```

Comment:
nit: Should it be 3.21 i.e. `# Protobuf >3.21`?

Reply:
This is an older problem we saw earlier this year, where the docker images automatically upgraded to 3.21, because protobuf version wasn't constrained, which resulted in runtime errors caused by a mismatch in the protobuf version installed and the version used in the build of the framework.

I do see that `>3.20` is misleading, though. It should be `>=3.21`. I'll correct this. Thanks!